src/main.c

```diff
@@ -58,15 +63,10 @@ static void do_startup(GObject *object, gpointer data) {
 
 static void do_activate(GObject *object, gpointer data) {
```

Comment:
Can update this to `do_activate(GtkApplication *app)` which will make the following code a little more readable.